storage/proto/backend.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
```

Comment:
Is this short term until we prove it works or long term home for the pb?  I would have imagined that one would find the API pb in the opi-api repo.

Reply:
Yes, short term
I will move `proto` folder it to near the spec here https://github.com/opiproject/opi-api/blob/main/storage/spec.md

Reply:
Yes, long-term, this does indeed move out to the `opi-api` repo.